lib/utils/components/profile_tap.dart

```diff
@@ -1,41 +1,138 @@
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return GestureDetector(
-      onTap: (){},
-      child: Row(
-        children: [
-          Padding(
-            padding: EdgeInsets.all(10.0),
-            child: ClipRRect(
-              borderRadius: BorderRadius.circular(20),
-              child: Image.asset(imagePath),
+    return DropdownButtonHideUnderline(
+      child: DropdownButton2(
+        customButton: Row(
+          children: [
+            Padding(
+              padding: EdgeInsets.all(10.0),
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(20),
+                child: Image.asset(imagePath),
+              ),
+            ),
+            Text(
+              profileName,
+              style: TextStyle(
+                color: AppColors.lightGreyColor,
+                fontSize: 18,
+              ),
+            ),
+          ],
+        ),
+        items: [
+          ...MenuItems.firstItems.map(
+            (item) => DropdownMenuItem<MenuItem>(
+              value: item,
+              child: MenuItems.buildItem(item),
             ),
           ),
-          Text(
-            profileName,
-            style: TextStyle(
-              color: AppColors.lightGreyColor,
-              fontSize: 18,
+          const DropdownMenuItem<Divider>(
+              enabled: false,
+              child: Divider(
+                color: AppColors.yellowColor,
+              )),
+          ...MenuItems.secondItems.map(
+            (item) => DropdownMenuItem<MenuItem>(
+              value: item,
+              child: MenuItems.buildItem(item),
             ),
           ),
         ],
+        onChanged: (value) {
+          MenuItems.onChanged(context, value as MenuItem);
+        },
+        dropdownStyleData: DropdownStyleData(
+          width: 160,
+          padding: const EdgeInsets.symmetric(vertical: 6),
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(4),
+            color: AppColors.darkGreyColor,
+          ),
+          elevation: 8,
+          offset: const Offset(40, -4),
+        ),
+        menuItemStyleData: MenuItemStyleData(
+          customHeights: [
+            ...List<double>.filled(MenuItems.firstItems.length, 48),
+            8,
+            ...List<double>.filled(MenuItems.secondItems.length, 48),
+          ],
+          padding: const EdgeInsets.only(left: 16, right: 16),
+        ),
       ),
     );
   }
-}
\ No newline at end of file
+}
+
+class MenuItem {
+  final String text;
+  final IconData icon;
+
+  const MenuItem({
+    required this.text,
+    required this.icon,
+  });
+}
+
+class MenuItems {
+  static const List<MenuItem> firstItems = [like, share];
+  static const List<MenuItem> secondItems = [cancel];
+
+  static const like = MenuItem(text: 'Indstillinger', icon: Icons.settings);
+  static const share = MenuItem(text: 'FAQ', icon: Icons.share);
```

Comment:
```suggestion
  static const share = MenuItem(text: 'FAQ', icon: Icons.share);
  ???Icons.share???
```

skal ikke være share